Bottom: f2fb0b848eca4756c49653a4b836b878b0c8a9a0
Top:    6fb99b94d9f16c9d04c4593bd468a52e5be033ae
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-03 02:36:34 +0000

Refresh of improve-gl-stuff

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index f4367ce..a55ca26 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -394,7 +394,6 @@ hidgl_draw_arc (double width, int x, int y, int rx, int ry,
 
   hidgl_ensure_triangle_space (&buffer, 2 * slices);
 
-
   if (delta_angle < 0) {
     start_angle += delta_angle;
     delta_angle = - delta_angle;
